NEWS.md

```diff
@@ -8,6 +8,28 @@
 ## Updates of Existing Functions
 
 ## Breaking Changes
+  
+- The following function arguments are entering the next phase of the deprecation process: (#2487)
+
+### Phase 1 (warning)
+
+### Phase 2 (error)
+
+### Phase 3 (removed)
+  - `consolidate_metadata(check_keys)`
+  - Removed at v1.1.1 `compute_egfr(wt)`
```

Comment:
Did this one get updated??  I don't see it changed in the list of files.   `compute_kidney.R`

Reply:
Or did we miss it in the last release?

Reply:
In the repo: https://github.com/pharmaverse/admiral/blob/2487-general-issue-move-deprecated-arguments-and-functions-to-next-phase-or-remove/R/compute_kidney.R
There is no `wt` argument anymore. 
This is the case for a few other arguments down the line of the list in news.md (indicated by me with: `Removed at v1.1.1`)

Reply:
Ah so we missed these!  Thanks for clarifying and be specific in the news section.